registry/mainnet/interchain/squid.tokenlist.json

```
@@ -6171,6 +6171,39 @@
         }
       ],
       "coinGeckoId": ""
+    },
+    "0x73c6c46c441ee16932b99375a35d2d42c5a41054b86901fb86e285d6b9128154": {
+      "tokenId": "0x73c6c46c441ee16932b99375a35d2d42c5a41054b86901fb86e285d6b9128154",
+      "deployer": "rHgMYhM946gg9QnWYHU7uPgrBFt4PLjy4a",
+      "originalMinter": null,
+      "prettySymbol": "USDC.xrpl",
+      "decimals": 15,
+      "originAxelarChainId": "xrpl",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
+      "coinGeckoId": "usd-coin",
+      "iconUrls": {
+        "svg": "https://raw.githubusercontent.com/axelarnetwork/axelar-configs/main/images/tokens/usdc.svg"
+      },
+      "deploymentMessageId": "0xd60f6e328c778a4db5ed125842029f01ee3c91562486c38f3f34222e40be71e0-1929109",
+      "chains": [
+        {
+          "symbol": "USDC.xrpl",
+          "name": "USDC",
```

```suggestion
          "name": "USD Coin",
```
this should be the ERC20 token name, not the symbol. Since the token was already deployed, can't change it anymore on-chain, but in the future we can stick to the convention. We can still use USD Coin in the config if we want